From b3382b844bdb6a30e466f30cc1bb09770aa5a33d Mon Sep 17 00:00:00 2001
From: Daniel Vetter <daniel.vetter@ffwll.ch>
Date: Wed, 10 Jul 2013 17:51:10 +0200
Subject: drm: rip out dev->last_checked

Only ever re-cleared in drm_setup, otherwise completely unused.

Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Signed-off-by: Dave Airlie <airlied@gmail.com>
(cherry picked from commit 23367ff49065505e4a255dba2117f654ca26063f)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/gpu/drm/drm_fops.c | 1 -
 include/drm/drmP.h         | 1 -
 2 files changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/drm_fops.c b/drivers/gpu/drm/drm_fops.c
index 2f86ccb86dfc..8b3eac08b48d 100644
--- a/drivers/gpu/drm/drm_fops.c
+++ b/drivers/gpu/drm/drm_fops.c
@@ -77,7 +77,6 @@ static int drm_setup(struct drm_device * dev)
 
 	dev->context_flag = 0;
 	dev->last_context = 0;
-	dev->last_checked = 0;
 	dev->if_version = 0;
 
 	dev->buf_async = NULL;
diff --git a/include/drm/drmP.h b/include/drm/drmP.h
index a47fec92c6ca..f8ad91bc0f94 100644
--- a/include/drm/drmP.h
+++ b/include/drm/drmP.h
@@ -1130,7 +1130,6 @@ struct drm_device {
 	/*@{ */
 	int irq_enabled;		/**< True if irq handler is enabled */
 	__volatile__ long context_flag;	/**< Context swapping flag */
-	int last_checked;		/**< Last context checked for DMA */
 	int last_context;		/**< Last current context */
 	/*@} */
 
-- 
1.8.5.rc3

